src/test/resources/prepare-for-repackage.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
surprised to find this file here? Why is it here?

Reply:
It is used by the repackaging script. I could try to inline it in that script though.